packages/db/src/runtime/index.ts

```diff
@@ -16,7 +16,8 @@ export { sql };
 export type SqliteDB = LibSQLDatabase;
 export type { Table } from './types.js';
 export { createRemoteDatabaseClient, createLocalDatabaseClient } from './db-client.js';
-export { seedLocal } from './queries.js';
+// only used for in-memory db in test fixtures
+export { recreateTables } from './queries.js';
```

Comment:
Isn't it worth investigating if we can pull this function using dynamic import?

Reply:
Good catch, this can be removed now that in-memory dbs are no longer used